test/telemetry-test.ts

```diff
@@ -29,22 +29,22 @@ describe("telemetry", () => {
 
   it("sends data", async () => {
     Telemetry._instance = new Telemetry(noopEffects);
-    Telemetry.record({event: "build", step: "start", test: true});
```

Comment:
This reminded me why I annotated these as `test: true`: in case these events ran in CI and made it into our production events, I wanted an easy way to filter them out. Unsure if that's a good enough reason to keep it this way or not or simply try to firewall off any chance of them showing up in our actual data.

I did just double check and we have no recorded events with `test: true` in them.